tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -95,6 +95,11 @@ await Tracer.Instance.TracerManager.AgentWriter.FlushTracesAsync()
 
         try
         {
+            if (exception != null && scope is { Span: var span })
```

Comment:
Is there a test for this? That we are indeed setting the exception to a span?

Reply:
First, I don't think unit test is needed. I think this is more on integration tests.
But then I don't want to further complicate the integration cases we have here.... I can try to see if we can add a case but I wouldn't treat that as a blocker. 
However, this inspired me to add one case on my re-designed integration test suites. I'll share more about that later.

Reply:
Mmm, I'd say it wouldn't hurt to have a simple integration tests which throws an error and the snapshot has it included! I'll approve, but I do think we need it, consult with the .NET team about it

Reply:
I agree, this should definitely show up in the snapshots, especially as we already have multiple handlers that throw exceptions there 🤔 tbh, the fact it's not appearing implies it's not working to me 🤔

Reply:
Updated the integration tests accordingly. Resolving this.

Reply:
Thanks for the update, it looks like now we'll use the error information in the headers and put them on the spans like the handler would